Hypermedia.AspNetCore.Siren/HypermediaResourceFilter.cs

```diff
@@ -33,23 +40,27 @@ public void OnResultExecuting(ResultExecutingContext context)
                 .OfType<IPartialResource>()
                 .Select(filter => filter.PartialResource);
```

Comment:
We could extract this into a separate function for better testability.

Reply:
For the moment I extracted this into a private method.